### DISCUSSION TITLE
Download error (35) SSL connect error [https://conda.anaconda.org/conda-forge/noarch/repodata.json.zst] Send failure: Connection was reset

### DISCUSSION THREAD

**Discussion by tzehuey85** [opengeos/segment-geospatial#250] [Q&A]
Hi everyone

after I tried to install "mamba install arcpy deep-learning-essentials segment-geospatial pygis -c esri -c conda-forge", I received the above mentioned error. How can I get sole this issue?

**Comment by giswqs** [comment]
This is likely your network issue. Try using mamba to install any packages such as numpy and pandas. If it doesn't work, then it is either a mamba issue or your network issue

**Reply by tzehuey85** [reply]
I tried to install numpy using mamba, failed. the error show 'mamba' is not recognized as an internal or external command,
operable program or batch file.
